schema/default_library.json

```diff
@@ -302,5 +302,29 @@
       "illuminance_setpoint": 500,
       "height": 3
     }
+  ],
+  "pitched_roof_types": [
```

Comment:
@shorowit i am thinking if this should be "pitched_roofs" since you can have two different roofs of the same type, i know we may have waffled on this before, but that is how it seems to me now.  Also, in the space edit view the dropdown column says "Pitched Roof" which seems more correct than "Pitched Roof Type" in context of the other things to edit like "Thermal Zone", etc